packages/quantic/force-app/main/default/lwc/quanticSearchBox/quanticSearchBox.html

```diff
@@ -34,7 +34,7 @@
                   onclick={clearInput}
                   icon-name="utility:close"
                   variant="bare"
-                  class="slds-input__icon slds-input__icon_right input__icon_cursor"
+                  class="slds-input__icon slds-input__icon_right input__icon_cursor slds-grid"
```

Comment:
slds-grid adds `display: flex` which fixes the alignment issue using a SLDS class instead of adding a custom css class.